{"uuid": "cf7480fc-ad47-4267-9bc8-bb3a7d65465d", "children": ["da610adf-758b-42e7-b780-774dd46b247d"], "befores": [{"name": "login_user", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range\n", "trace": "  File \"D:\\study\\Diplom\\Diplom_2\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"D:\\study\\Diplom\\Diplom_2\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"D:\\study\\Diplom\\Diplom_2\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\study\\Diplom\\Diplom_2\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1125, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\study\\Diplom\\Diplom_2\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 887, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\study\\Diplom\\Diplom_2\\tests\\conftest.py\", line 23, in login_user\n    \"email\": create_user['login_pass'][0],\n             ~~~~~~~~~~~~~~~~~~~~~~~~~^^^\n"}, "start": 1719861146794, "stop": 1719861146794}], "start": 1719861146794, "stop": 1719861146848}